dts/arm/st/h7/stm32h723.dtsi

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020 Alexander Kozhinov
+ * AlexanderKozhinov@yandex.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/h7/stm32h7.dtsi>
+
+/ {
+	/* DTCM memory directly coppled to CPU */
+	dtcm: memory@20000000 {
+		compatible = "arm,dtcm";
+		reg = <0x20000000 DT_SIZE_K(128)>;
+	};
+
+	/* AXI SRAM in D1 domain (AXI bus) */
+	axi_sram: memory@24000000 {
+		reg = <0x24000000 DT_SIZE_K(128)>;
+		compatible = "mmio-sram";
+	};
+
+	/* RAM shared between ITCM and AXI in D1 (AXI bus) */
+	axi_itcm_sram: memory@24020000 {
+		reg = <0x24020000 DT_SIZE_K(192)>;
+		compatible = "mmio-sram";
+	};
+
+	/* SRAM1 in D2 domain (AHB bus) */
+	sram1: memory@30000000 {
+		reg = <0x30000000 DT_SIZE_K(16)>;
+		compatible = "mmio-sram";
+	};
+
+	/* SRAM2 in D2 domain (AHB bus) */
+	sram2: memory@30004000 {
+		reg = <0x30004000 DT_SIZE_K(16)>;
+		compatible = "mmio-sram";
+	};
+
+	/* SRAM4 in D3 domain (AHB bus) */
+	sram4: memory@38000000 {
+		reg = <0x38000000 DT_SIZE_K(16)>;
+		compatible = "mmio-sram";
+	};
```

Comment:
I'm still not sure if adding these RAMs is useful, not blocking though.

Reply:
They are implemented in the HW, so why not. The other question is, should we use it or not.

Reply:
Removed following blocks:

> /* SRAM4 in D3 domain (AHB bus) */
	sram4: memory@38000000 {
		reg = <0x38000000 DT_SIZE_K(16)>;
		compatible = "mmio-sram";
	};

> /* Backup SRAM in D3 domain (AHB bus) */
	backup_sram: memory@38800000 {
		reg = <0x38800000 DT_SIZE_K(4)>;
		compatible = "mmio-sram";
	};